### ISSUE TITLE
Package request: Astal

### ISSUE THREAD

**Issue by jpeeler** [solopasha/hyprlandRPM#21]
https://github.com/Aylur/Astal

Astal is the GTK4 version of AGS. Currently no tags or releases have been made, but I'm assuming it's probably working decently well already.

**Comment by solopasha** [comment]
Seems also needs gtk4-layer-shell. I'll look into it next week.

**Comment by michaelhaaf** [comment]
On a related note, are there plans for this copr to support [ags v2.0](https://github.com/Aylur/ags/releases/tag/v2.0.0)? 

I see there has been a commit about this, and that it was [reverted](https://github.com/solopasha/hyprlandRPM/commit/57360f95e6e3d8bc0ac55be552e5dbbc67828159). I imagine it's not a straightforward update, and there aren't non-go-fedora build instructions in the [ags documentation](https://aylur.github.io/ags/guide/install.html) that I'm aware of.

**Comment by solopasha** [comment]
Yes, I made some initial builds in https://copr.fedorainfracloud.org/coprs/solopasha/astal. Could you try them?

**Comment by michaelhaaf** [comment]
Thanks. I haven't had the time to test ags2/astal on a live system yet, but your other copr builds successfully on my [BlueBuild-based](https://blue-build.org/) system: (https://github.com/michaelhaaf/bluebuild-custom/actions/runs/11871978593/job/33085130839)

**Comment by michaelhaaf** [comment]
Tested running `ags init` using the https://copr.fedorainfracloud.org/coprs/solopasha/astal copr on a live system today (Fedora 41). 

So far run into the same issue described here, mostly by arch users. https://github.com/Aylur/ags/issues/617 . 

Haven't decided on the way to proceed yet with these undeclared node dependencies from upstream.
